templates/components/success-story.html

```diff
@@ -13,15 +13,15 @@ <h2 class="widget-title"><span aria-hidden="true" class="icon-success-stories"><
                                 <a href="{{ story.get_absolute_url }}">{{ story.pull_quote }}</a>
                             </blockquote>
 
-                            <table cellpadding="0" cellspacing="0" border="0" width="100%" class="quote-from">
+                            <table class="quote-from">
                                 <tbody>
                                     <tr>
                                         {% comment %}
                                         This image tag should be for a Company logo, if they have supplied one
                                         Currently this path is for a static logo sample, which no longer exists
                                         <td><a href="{{ story.get_absolute_url }}"><img src="{{ STATIC_URL }}img/ILM_logo.png"></a></td>
                                         {% endcomment %}
-                                        <td><p><a href="{{ story.get_absolute_url }}">{{ story.name }}</a> <em>by {{ story.author }}</em></p></td>
+                                        <td><p><a href="{{ story.get_absolute_url }}">aaa{{ story.name }}</a> <em>by {{ story.author }}</em></p></td>
```

Comment:
Oops.. thanks for catching. I'll make another commit right away. Sorry about this.